contracts/test/CompoundGovernor.t.sol

```diff
@@ -33,5 +35,149 @@ contract CompoundGovernorTest is Test, CompoundGovernorConstants {
         assertEq(address(governor.timelock()), TIMELOCK_ADDRESS);
         assertEq(address(governor.token()), COMP_TOKEN_ADDRESS);
         assertEq(governor.owner(), owner);
+        assertEq(governor.whitelistGuardian(), whitelistGuardian);
+        (address _proposalGuardian, uint96 _expiration) = governor.proposalGuardian();
+        assertEq(_proposalGuardian, proposalGuardian.account);
```

Comment:
I don't think that works. because governor.proposalGuardian() returns a tuple (address, uint256) not ProposalGuardian struct since we don't have a getter function, just a public state var.